Validation/RecoTrack/python/TrackValidation_cff.py

```diff
@@ -462,7 +462,8 @@ def _getMVASelectors(postfix):
     dodEdxPlots = False,
     doPVAssociationPlots = False,
     doSimPlots = False,
-    doResolutionPlotsForLabels = ["disabled"],
+#    doResolutionPlotsForLabels = ["disabled"],
+    doResolutionPlotsForLabels = [], # enable resolution plots for building as well
```

Comment:
Just to note that they were disabled in the standard sequence to save memory. Is the memory use in VALIDATION still a concern?

Reply:
which memory use problem ?

Reply:
Is this potential memory issue been addressed somehow?

Reply:
ciao
sorry, but can you please comment / point me to something about this "potential memory issue"
thanks

Reply:
@mtosi I am just referring to the comment from @makortel, who is very welcome to clarify his concern :-)

Reply:
Back in the "histogram per stream" days the memory use of VALIDATION was a concern, and IIRC MTV in total took ~70 MB (per stream). Most of the memory is needed for 2D histograms like the ones used for resolution plots. That's why they are disabled for most of the cases for the standard sequence (after all they are not really needed for release validation for all MTV instances and track collections), and are included in the standalone/tracking-only mode instead.

If DQM is fine for MTV using even more memory in VALIDATION, that's fine for me. Anyway tests should tell how large the increase is.